web/src/components/countrytableoverlayifnodata.js

```diff
@@ -26,6 +27,20 @@ const CountryTableOverlayIfNoData = ({ zoneTimeIndex }) => {
       <div className="no-data-overlay-background" />
       <div className="no-data-overlay-message">
         {__(isRealtimeData ? 'country-panel.noLiveData' : 'country-panel.noDataAtTimestamp')}
+        {!isRealtimeData ? null : (
+          <React.Fragment>
+            {'. '}
+            {'Help us identify the problem by taking a look at the '}
+            <a
+              href={`https://kibana.electricitymap.org/app/kibana#/discover/10af54f0-0c4a-11e9-85c1-1d63df8c862c?_g=(refreshInterval:('$$hashKey':'object:232',display:'5%20minutes',pause:!f,section:2,value:300000),time:(from:now-24h,mode:quick,to:now))&_a=(columns:!(message,extra.key,level),filters:!(('$state':(store:appState),meta:(alias:!n,disabled:!t,index:'96f67170-0c49-11e9-85c1-1d63df8c862c',key:level,negate:!f,params:(query:ERROR,type:phrase),type:phrase,value:ERROR),query:(match:(level:(query:ERROR,type:phrase)))),('$state':(store:appState),meta:(alias:!n,disabled:!f,index:'96f67170-0c49-11e9-85c1-1d63df8c862c',key:extra.key,negate:!f,params:(query:${zoneId},type:phrase),type:phrase,value:${zoneId}),query:(match:(extra.key:(query:${zoneId},type:phrase))))),index:'96f67170-0c49-11e9-85c1-1d63df8c862c',interval:auto,query:(language:lucene,query:''),sort:!('@timestamp',desc))`}
```

Comment:
Could we use a brighter color for this link? It's a little bit hard to read from the screenshot you provided:

![image](https://user-images.githubusercontent.com/1655848/81564393-f3ed9580-9397-11ea-9a14-cd43dc0b14f3.png)

Reply:
what about this

![image](https://user-images.githubusercontent.com/1655848/81569887-27342280-93a0-11ea-8fe9-9e14be4fca10.png)

Reply:
Looks better to me! Feel free to also add a thin border around the message box if you think it's an improvement (since the two backgrounds now have a very similar color):

![image](https://user-images.githubusercontent.com/1216874/81570393-d53fcc80-93a0-11ea-8403-110fc7349c94.png)